include/glow/Backends/DummyDeviceManager.h

```diff
@@ -44,6 +44,9 @@ class DummyDeviceManager : public DeviceManager {
   }
   bool isMemoryAvailable(uint64_t) const override { return true; }
 
+  /// Returns uninitialized DeviceInfo
+  DeviceInfo getDeviceInfo() const override { return DeviceInfo(); }
```

Comment:
Would it be better to just make this the default implementation instead of having that be a pure virtual?

Reply:
Agreed, thanks for the suggestion.

Reply:
The DeviceConfigs is passed into HostManager for initialization. Basically, it contains all backend related information and then the DeviceManagers are created. I don't think this function is necessary. The numbers used for roofline calculation should be added in to DeviceConfigs and passed to HostManager.

Reply:
Addressed by removing pure virtual function.